earth_enterprise/docker/README.md

```diff
@@ -0,0 +1,182 @@
+# Docker Image for Running GE Open Source Earth Server and Fusion Server
+
+## Setting up Docker
+
+### Ubuntu
```

Comment:
Minor: we should probably link to Docker instructions for other operating systems. This can be done later